### ISSUE TITLE
Mistake in pronounced of 9 in Mandarin Chinese

### ISSUE THREAD

**Issue by Giovani93** [espeak-ng/espeak-ng#257]
Hi!
I have noticed, that 9 is pronounced, as xiǔ, as a number, not character (logogram). Logogram for nine in chinese is pronounced corectly, as 'jiǔ', but normal arabic numerals not.
Please, could You fix It?
Thank You!
Marco Oros

**Comment by valdisvi** [comment]
As far as I understood, talk is about spelling Mandarin Chinese and Arabic presentation of numbers, e.g. 1 vs 一 or 9 vs 九. Question is, how it should be pronounced, and what about other numbers?

**Comment by Giovani93** [comment]
Another numbers are pronounced corectly, I think on single arabic numerals, from 0 to 9. But, 9 (nine) is pronounced as xiǔ, not jiǔ as a chinese numeral, which is pronounced, ofcourse, corectly. Also, I don't know espeak phoneme transcription, but those numbers are written in pinyin, as I wrote and Pinyin is pronounced also good in Mandarin. This is for information and also, I can't instal github version of Espeak, I use Espeak with NVDA. This is also just for information, so I must translate those chinese characters to numbers.

**Comment by valdisvi** [comment]
Is [this pronunciation](http://odo.lv/ftp/espeak/.0A63.wav) correct?

**Comment by Giovani93** [comment]
No. Try enter a chinese character for nine and describe It phonetically in Espeak. After that, edit It in dictionary, maybe It is in chinese list mistake.

**Comment by valdisvi** [comment]
So, is [this](http://odo.lv/ftp/espeak/.45D6.wav) correct?

**Comment by Giovani93** [comment]
No!
Try check this dictionary for character '九' and 9 pronunciation by 
searching. Maybe good idea is to use search box.

**Comment by Giovani93** [comment]
Really, I don't know, how to help You.

**Comment by Giovani93** [comment]
Also, I have idea.
Espeakedit have also function to write and translate those letters to 
the chinese characters and another letters too.
That's a pity, that maybe We haven't enough people, which can speak 
Mandarin Chinese, but I know only little. I am interested on learning 
languages.
Thank You.

**Comment by valdisvi** [comment]
[Here are three words](http://odo.lv/Espeak?eslang=-vzh&speed=160&output=-w&debug=-x&text=9+%E4%B9%9D+%5B%5BtS%3B%27iou%5D%5D&formmethod=get) for 9 九 and espeak-ng phonemes.
Should espeak-ng say **9** as third word in provided form?
To provide some other pronunciation, you can write some other numbers/words and provide link to form with _Show data in address_ option.

**Comment by rhdunn** [comment]
The `九` character has the pronunciation `jiu3` (line 861 of `zh_list`). This is correct according to the `kMandarin` reading in the Unihan database (http://www.unicode.org/cgi-bin/GetUnihanData.pl?codepoint=4E5D). There are several issues here:

1.  `9` should be pronounced like `tS;'iou` with tone 3, to match the mandarin reading of `九`;
2.  the pronunciations of the Han characters are using number to represent tones (as in the pinyin transliteration of Mandarin), but those numbers are getting interpreted as actual numbers (so you get a reading like `9 3` for `九`);
3.  the `tS;` phoneme sounds wrong -- it sounds to me like a `k` instead of a `tS`-like sound (the `tS;h` phoneme sounds closer, but should be used with less aspiration).

For item 1, I also have the https://github.com/espeak-ng/espeak-ng/issues/83 task. That is, the numbers should be written as:

    _9  九
    _9X  九十
    ...

i.e. mapping the arabic numerals to the appropriate language written form, then pronounced using the language rules for those words/characters. That will make it easier to get the correct pronunciation and better support accent differences.

**Comment by Giovani93** [comment]
I agree with You.
But, nine and chinese nine are pronounced differently. Please, check 
also our nine '9' in chinese list, or rules file.

**Comment by materia-cccp** [comment]
It's 2019 and it's still not fixed I'm gonna pr
